sign/tbls/tbls_test.go

```diff
@@ -15,7 +16,7 @@ func TestTBLS(test *testing.T) {
 	suite := bn256.NewSuite()
 	n := 10
 	t := n/2 + 1
-	secret := suite.G1().Scalar().Pick(suite.RandomStream())
+	secret := suite.G2().Scalar().Pick(suite.RandomStream())
```

Comment:
Are you sure? I didn't look into the code, but usually you need to 'mix' G1 and G2.